server/controllers/duels.js

```diff
@@ -391,11 +391,20 @@ function createShuffledQuestionTypesArray() {
  */
 async function getDuel(id, username) {
   const res = await queryPromise("CALL getDuel(?,?);", [id, username]);
-
   if (res[0].length === 0) {
     return null;
   }
-  return formatDuel(res[0], username);
+
+  const TTL = await (async function () {
+    const res = await queryPromise(
+      "SELECT `value` FROM `server_informations` WHERE `key` = 'config_duel_rounds_per_duel';"
+    );
+    return res[0].value;
+  })();
```

Comment:
It it possible to have the two calls running in parallel with an `Promise.all()` to avoid increasing the time to request a duel?

Reply:
Let's do this

Reply:
But after the réunion

Reply:
You can also do it in just one query!

Reply:
That's the query you already commented @FrancoisPog :)

Reply:
Yep but maybe this is even better 
```js
const res = await queryPromise("CALL getDuel(?,?); SELECT `value` FROM `server_informations` WHERE `key` = 'config_duel_rounds_per_duel';");
```

Reply:
Oh okay! Sure